### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by anut123**

### CODE REVIEW

File: domains.yml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+   - utter_ask_location
+   - utter_location_denied   
+   - utter_please_try_after_sometime
+   - action_show_hospitals
+   - utter_ask_for_room_type
+   - utter_ask_for_appointment_booking 
+   - utter_ask_time_slots 
+   - action_taking_location   
+
+templates:
```
Comment (SanjayMarreddi) [inline domains.yml:65]: I think this should be "responses" instead of "templates" ?

**Review by SanjayMarreddi — CHANGES_REQUESTED** [review]
Also the file should be "domain.yml" instead of "domains.yml" .

Finally I will merge this code after the completion the action.py file because we still need to include some info in this domains.yml file from actions.py like slots,etc..

**Comment by anut123** [comment]
No sir but in rasa video they wrote templates

**Comment by SanjayMarreddi** [comment]
@anut123  Yeah you are Correct ! 
It seems previous versions used "responses". Now "templates" is being used .

**Pull request closed without merging**